### PR TITLE
Download the full test report

### PR DESCRIPTION
**Pull request by tertsdiepraam**
As promised in https://github.com/uutils/coreutils/pull/3134 :)

Here's hoping that it's going to work...

### CODE REVIEW

File: .github/workflows/gnu-data.yml [hunk 1]
```diff
@@ -58,6 +66,10 @@ jobs:
        cat merged.json
        mv merged.json size-result.json
 
+       echo "== dl/gnu-full-result.json =="
+       cat dl/gnu-full-result.json
+       mv dl/gnu-full-result.json gnu-full-result.json
```
Comment (sylvestre) [inline .github/workflows/gnu-data.yml:71]: it is just going to download it and not do anything with it. I guess if it expected :)

Comment (tertsdiepraam) [reply .github/workflows/gnu-data.yml:71]: Yeah, all the processing is done in the browser, so the file just needs to be available. But maybe there's a better way to access files uploaded as artifacts if they don't need any processing?

Comment (sylvestre) [reply .github/workflows/gnu-data.yml:71]: well, it won't be available, it needs to be committed now ?

Comment (tertsdiepraam) [reply .github/workflows/gnu-data.yml:71]: Is it not added to the "Update of the data" commit below?

Comment (sylvestre) [reply .github/workflows/gnu-data.yml:71]: Let see :)

**Comment by sylvestre** [comment]
Failed:
```
Run dawidd6/action-download-artifact@v2
  with:
    workflow: CICD.yml
    name: gnu-full-result.json
    repo: uutils/coreutils
    path: dl
    github_token: ***
    workflow_conclusion: success
==> Workflow: CICD.yml
==> Repo: uutils/coreutils
==> Conclusion: success
==> RunID: 1844504688

Error: no artifacts found
```

**Comment by tertsdiepraam** [comment]
Is there any good way to debug this?

**Comment by sylvestre** [comment]
Commit directly in the repo :)

**Pull request merged**